src/components/Input/styles/InputStyles.js

```diff
@@ -24,6 +24,10 @@ const InputStyles = styled.input`
     border-color: ${get('colors.border.medium')};
   }
 
+  &:required {
+    box-shadow: none;
```

Comment:
This removes the focus effect on Firefox as well by overriding the rule from line 22.

Reply:
Probably needs to be moved above that rule.

Reply:
Yeah, I changed the order. Other than that the fix does not change any existing behavior except for removing the red shadow on FF when required fields are initially displayed.